code/modules/clothing/glasses/hud.dm

```diff
@@ -71,6 +72,7 @@
 	name = "Ironhammer tactical HUD"
 	desc = "Flash-resistant goggles with inbuilt combat and security information."
 	icon_state = "swatgoggles"
+	style = STYLE_LOW
```

Comment:
Is this needed?
Inspector is supposed to be a Style user, this hits him too.
And the rest of ironhammer clothes are also low Style already so this won't even matter.

(Accidentally posted this as a review the first time around, ignore that)

Reply:
He has access to sunglasses in the inspector office, and can take more through loadout.

Reply:
Tho honestly sechud lenses would be cool